src/test/components/TooltipMarker.test.js

```diff
@@ -678,9 +678,7 @@ describe('TooltipMarker', function() {
     `);
 
     // Add another thread with the thread Id we are going to refer from the marker.
-    profile.threads[1] = getEmptyThread();
-    profile.threads[1].name = 'Renderer';
-    profile.threads[1].tid = threadId;
+    profile.threads.push(getEmptyThread({ name: 'Renderer', tid: threadId }));
```

Comment:
Not necessary, but I did it just to check that this was working (because I did it elsewhere and it didn't work...). I think it's slightly better but I can revert if you prefer.

Reply:
It looks better to me too!